src/workloads/query/PercentilesWindow.yml

```diff
@@ -0,0 +1,188 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This tests $percentile window functions over various bounded window sizes. Bounded window
```

Comment:
This comment might be confusing because we actually use window function with an accurate algorithm for computing over _unbounded_ removable windows. The list that follows also seems to imply that range windows aren't removable (because you differentiate "small removable windows" vs "small range windows"). I'd suggest "1. Small document-based removable windows" and "4. Small range-based removable windows"